server/sonar-web/src/main/js/apps/marketplace/App.tsx

```diff
@@ -148,6 +150,19 @@ export default class App extends React.PureComponent<Props, State> {
       () => {}
     );
 
+  fetchEditionStatus = () =>
```

Comment:
there is something wrong with loading here: the flag is not always set, and it's actually not used in render

Reply:
Ah yeah I forgot to use it in render, I'll fix it.
As for not every `fetchSomething` function setting it, I think some of them should not, like `fetchEditionStatus` that wont use for sure because I'm gonna change things to refresh the status every 5 seconds.
And we could discuss about `fetchPendingPlugins`, I'm not sur this one really deserve a loading flag. At page loading we will display the pending plugins as an alert if there are some, no need to notify the user that we are fetching that... And when clicking an install button there is already the spinner of the current install before the pending actions are displayed...